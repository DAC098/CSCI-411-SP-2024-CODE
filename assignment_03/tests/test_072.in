2 10
46 211
9395
5319
3991
8652
8080
5739
9570
6183
914
6666
