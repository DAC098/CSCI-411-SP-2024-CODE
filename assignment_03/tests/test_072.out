7 43 
56 13 
73 3 
78 24 
0 0 
111 3 
153 12 
0 0 
0 0 
44 22 
